### ISSUE TITLE
Add Programmatic Labels to "Investigator" and "Senior Official" Form Fields

### ISSUE THREAD

**Issue by SamiyahKey** [cisagov/manage.get.gov#3181]
### Story

“Investigator” and “Senior official” form fields are visually presented on the page, but they are not programmatically marked up. This means that screen reader users cannot properly interpret with these fields because they lack the necessary programmatic labels. The two form fields should be programmatically labeled so that users who rely on screen readers can understand the purpose of each field, fill them out correctly, and navigate through the form effectively. Without programmatic labeling, screen readers will not be able to understand what information needs to be filled in the form field making it difficult for AT users.  

As a screen reader user,
I need the "Investigator" and "Senior Official" form fields to have proper programmatic labels,
So that I can understand the purpose of each field, fill them out accurately, and navigate the form without confusion.

URL: career-program.gov | Change domain request
https://getgov-staging.app.cloud.gov/admin/registrar/domainrequest/284/change/

### Acceptance Criteria

Programmatic Labels:

Investigator Field:

- [x] Ensure ANDI output state "Investigator" for the combo box is " Investigator, combo-box, collapsed, edit, has autocomplete. To set the value, use the arrow keys or type the text."

Senior Official Field:

- [x] Ensure ANDI output state "Senior official" for the combo box is "Senior official, combo-box, collapsed, edit, has autocomplete. To set the value, use the arrow keys or type the text."

Testing Requirements:

- [x] Verify that screen readers (NVDA, JAWS, VoiceOver) announce the correct label for each form field.

- [x] Ensure that users can navigate and interact with the form fields seamlessly using only a keyboard and screen reader.

**Comment by SamiyahKey** [comment]
AC's have been updated @CocoByte , let me know if you have questions!

**Comment by Katherine-MN** [comment]
Removed "skipped design" to reflect that an update to design documentation or libraries is not needed.
